packages/gatsby-recipes/src/providers/gatsby/plugin.test.js

```diff
@@ -20,6 +20,18 @@ const HELLO_WORLD_FIXTURE = path.join(
 )
 const name = `gatsby-source-filesystem`
 
+// Some of these are slow tests, because they hit the network
+jest.setTimeout(20000)
+
+function normalizePlugin({ _message, id, name, options }) {
+  return {
+    _message,
+    id,
+    name,
+    options,
+  }
+}
```

Comment:
I've added a jest way below but I'm fine with this approach. I would like to swap the allowlist with a deny list. We remove readme & description from the snapshot. We want the snapshot to fail if new fields are being added.

Reply:
Do we though? New fields aren't a breaking change.

Reply:
Not breaking but it's something you want to see in your snapshots. Without it, it's the same as testing with toEqual.